scripts/validate-build.js

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
+// eslint-disable-next-line strict
+'use strict';
+
+const fs = require('fs');
+const esprima = require('esprima');
+const walk = require('esprima-walk');
+const colors = require('colors');
```

Comment:
Let's make sure these packages are in our dev dependencies. Looks like `colors` is a dependency of a dependency.